app/lib/common/utilities/email_sender.dart

```diff
@@ -62,9 +69,29 @@ class EmailRoute extends StatelessWidget {
                   onPressed: () {
                     if (formKey.currentState.validate()) {
                       print('valid');
-                      email.send(bodyController.text, subjectController.text,
-                          recipientController.text);
+                      //email.send(bodyController.text, subjectController.text,recipientController.text, pdf);
+                      Future.value(email.send(
```

Comment:
As we saw earlier, `send` shouldn't have a future return type. You can probably just launch showDialog as asynchronous and it will appear when the function is done. If you want it blocking interaction, you'll need to use `barrierDismissible` like `loading_screen_overlay.dart`. 

- [ ] The changes to the return type should force changes here

Reply:
This is what I initially wanted but since send is async function it has to have Future as return type. Also when I changed it to String, dialog box occurs before gmail launch.